packages/config/config/devices/0x010f/fgt001.json

```diff
@@ -18,6 +18,15 @@
 		"max": "255.255"
 	},
 	"supportsZWavePlus": true,
+	"compat": {
+		"commandClasses": {
+			"remove": {
+				"0x80": {
+					"endpoints": [0]
+				}
+			}
+		}
+	},
```

Comment:
The `"compat"` section should be moved to the end of the file (before "metadata" if that exists")